ios/Classes/BetterPlayerPlugin.m

```diff
@@ -85,6 +86,36 @@ - (void)onPlayerSetup:(BetterPlayer*)player
     result(@{@"textureId" : @(textureId)});
 }
 
+- (void)addObservers:(BetterPlayer*) player {
```

Comment:
`BetterPlayer`'s `addObservers` is called from 2 places.

- `setDataSourcePlayerItem`
- `updatePlayingState`

`BetterPlayerPlugin`'s `addObservers` is called only before calling `setDataSourceURL`.
Is it enough? Please check it 🙇

Reply:
@noboru-i 
This is my opinion,

Because `BetterPlayer` call `removeObservers` in `itemDidPlayToEndTime`, and actually we can replay or seek video after playback ended, so it must check and call `addObservers` every time the player `updatePlayingState`.

But in `BetterPlayerPlugin`, i only call `removeObservers` at the start of `setDataSource` and on `dispose`,
so while the new data source is not set, we don't need to call `addObservers` again.

Reply:
Thanks. I got it. 👍